lib/handler.js

```diff
@@ -113,14 +118,17 @@ module.exports = function(options) {
         };
       }
 
-      return esclient
-        .bulk(_.defaults({ body: parsedEvent.actions }, bulkOpts))
-        .then(result => {
-          return {
-            result,
-            meta: parsedEvent.meta
-          };
-        });
+      return promiseRetry(retry => {
+        return esclient
+          .bulk(_.defaults({ body: parsedEvent.actions }, bulkOpts))
+          .then(result => {
+            return {
+              result,
+              meta: parsedEvent.meta
+            };
+          })
+          .catch(retry);
+      }, { retries: retryCount });
```

Comment:
You could pass along `retryOptions` with default `retryCount` instead so that consumer of library has access to other retry variables